lib/registry/registry-client.ts

```diff
@@ -0,0 +1,72 @@
+/**
```

Comment:
This file is really only about image manifests, I feel `manifests.ts` is a more appropriate file name.